misc/debug-adapter/modules/debug-adapter-core/src/main/java/org/ballerinalang/debugadapter/JBallerinaDebugServer.java

```diff
@@ -142,6 +142,7 @@
     private ThreadReferenceProxyImpl activeThread;
     private SuspendedContext suspendedContext;
     private boolean terminationRequestReceived = false;
+    private StackFrame topStackFrame;
```

Comment:
As this is basically contains the PREVIOUS breakpoint location information, shall we use a more meaningful name which indicates that?

Reply:
Changed the name to `previousTopFrame`

Reply:
Using the `topFrame` might still be misleading, when considering why we are using it. How about `lastLocationInfo`?

Reply:
Changed to `lastDebbugHitFrame`